assets/app/views/browse/edit-build-config.html

```diff
@@ -0,0 +1,257 @@
+                            </label>
+                          </div>
+                        </div>
+
+
+                        <div class="form-group">
+                          <label for="outpuTo">Output to</label>
+                          <div class="output-to-selects multiselectors">
+                            <div class="label-with-selector">
+                              <label class="multiselector-label">Namespace</label>
+                              <select 
+                                class="form-control multiselector select-builder-namespace"
+                                ng-model="options.pickedPushToNamespace"
+                                ng-options="name for name in pushTo.projects track by name"
+                                ng-change="updateImageStreams(options.pickedPushToNamespace, false, false)">
+                              </select>
+                            </div>
+                            <span class="punctuation">/</span>
+                            <div class="label-with-selector">
+                              <label class="multiselector-label">Image Stream</label>
+                              <select
+                                class="form-control multiselector select-builder-name"
+                                ng-model="options.pickedPushToImageStream"
+                                ng-options="name as name for name in pushTo.imageStreams track by name"
+                                ng-change="updateTags(options.pickedPushToImageStream)">
+                              </select>
+                            </div>
+                            <span class="punctuation">:</span>
+                            <div class="label-with-selector">
+                              <label class="multiselector-label">Tag</label>
+                              <select
+                                class="form-control multiselector select-builder-tag"
+                                ng-model="options.pickedPushToImageStreamTag"
+                                ng-options="name as name for name in pushTo.tags[options.pickedPushToImageStream] track by name">
+                              </select>
+                            </div>
+  <!--                           <span class="help action-inline">
+                              <a href data-toggle="tooltip" data-placement="right" data-original-title="This image can build.">
+                              <i class="pficon pficon-help"></i>
+                              </a>
+                            </span> -->
+                          </div>
+                        </div>
+                      </dl>
+                    </div>
+
+                    <div class="section">
+                      <h3>Environment Variables<span class="help action-inline">
+                        <a href data-toggle="tooltip" data-placement="right"
```

Comment:
yep they do, if we leave the the icon wrapped in the anchor and user clicks on the anchor the tooltip will still be visible, but if the icon is place outside the anchor, the tooltip will be hidden the moment the curcor leaves the anchor, which I guess is the desired case. right ?
If so I can update the logic on all other places :)